src/aliases/manager/service.go

```diff
@@ -0,0 +1,25 @@
+package aliasservice
```

Comment:
It seems you don't need this `manager` folder in this PR

Reply:
I guess you're right. I shouldn't have commited it, it was just to confirm that my code was loading well.
I'll remove it and the lines from `src/app.go`